examples/website/3d-tiles/app.tsx

```diff
@@ -35,6 +35,21 @@ export const INITIAL_VIEW_STATE = {
   zoom: 3 // Start zoomed out on US, tileset will center via "fly-to" on load
 };
 
+const Stats = styled.div`
```

Comment:
Ideally we have better defaults in the stats widget itself.

Reply:
The problem is that default position set to 'fixed' as I figured and somehow the whole docusaurus container takes this as a rule, not the parent position even when it sets through ref container, so the solution with overwriting styles without logic changing was quick :)

Reply:
as I found in docs in probe.gl we can overwrite css for header or info, but not for all container